dashboard/config/locales/en.yml

```diff
@@ -390,7 +390,7 @@ en:
       sprite_lab_documentation: "Sprite Lab Documentation"
       sprite_lab_tutorials: "Sprite Lab Tutorials"
       web_lab_documentation: "Web Lab Documentation"
-      project_beats_documentation: "Project Beats Documentation"
+      music_lab_documentation: "Music Lab Documentation"
```

Comment:
nit: can we make the `_documentation` entries alphabetical?

Reply:
yep!